plugins/@grouparoo/mongo/src/initializers/plugin.ts

```diff
@@ -41,7 +41,9 @@ export class Plugins extends Initializer {
         new TableSourceTemplate("mongo", { getTables, getColumns }),
         new TablePropertyTemplate("mongo"),
         new QueryPropertyTemplate("mongo"),
-        new QuerySourceTemplate("mongo"),
+        new QuerySourceTemplate("mongo", [
+          path.join(templateRoot, "query-source", "*.template"),
+        ]),
```

Comment:
The Mongo Query source needed a custom template as it requires "table"  ("collection") that the others don't